src/components/common/CitiesList/PopularCitiesList.jsx

```diff
@@ -0,0 +1,13 @@
+import usePopularCities from '../../../hooks/usePopularCities';
+import CitiesList from './CitiesList';
+
+const PopularCitiesList = ({ limit, increaseLimit }) => (
+  <CitiesList
+    limit={limit}
+    increaseLimit={increaseLimit}
+    showLoadMoreButton={false}
+    useCitiesHook={usePopularCities}
```

Comment:
You should not pass a hook as prop

https://stackoverflow.com/questions/66800263/passing-a-hook-as-a-prop